site/en/rerankers/rerankers-overview.md

```diff
@@ -0,0 +1,202 @@
+---
+id: rerankers-overview.md
+order: 1
+summary: PyMilvus model library integrates rerank functions to optimize the order of results returned from initial searches.
+title: Overview
+---
+
+# Overview
+
+In the realm of information retrieval and generative AI, a reranker is an essential tool that optimizes the order of results from initial searches. Rerankers differ from traditional [embedding models](embeddings.md) by taking a query and document as input and directly returning a similarity score instead of embeddings. This score indicates the relevance between the input query and document.
+
+Rerankers are often employed after the first stage retrieval, typically done via vector Approximate Nearest Neighbor (ANN) techniques. While ANN searches are efficient at fetching a broad set of potentially relevant results, they might not always prioritize results in terms of actual semantic closeness to the query. Here, rerankers is used to optimize the results order using deeper contextual analyses, often leveraging advanced machine learning models like BERT or other Transformer-based models. By doing this, rerankers can dramatically enhance the accuracy and relevance of the final results presented to the user.
+
+PyMilvus model library integrates rerank functions to optimize the order of results returned from initial searches. After you retrieved nearest embedings from Milvus, you can leverage these reranking tools to refine search results to enhance the precision of search outcomes.
+
+| Rerank Function | API or Open-sourced |
+| --------------- | ------------------- |
+| bgereranker     | Open-sourced        |
```

Comment:
Should we clarify that open-sourced models need to be downloaded while API ones require API keys?

Reply:
added a note 
![image](https://github.com/milvus-io/milvus-docs/assets/105278390/e9e76bfd-29c7-46b2-9b75-f4c467684024)